### ISSUE TITLE
Windows，training cann't launch，报错：'PYTHONPATH' 不是内部或外部命令，也不是可运行的程序

### ISSUE THREAD

**Issue by ultimatech-cn** [modelscope/facechain#99]
当执行训练后，后台报下面错误
```
'PYTHONPATH' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
训练已经完成！请切换至 [形象体验] 标签体验模型效果(Training done, please switch to the inference tab to generate photos.)
--------uuid:  qw
----------work_dir:  D:\AI\qw\personalizaition_lora
'PYTHONPATH' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
训练已经完成！请切换至 [形象体验] 标签体验模型效果(Training done, please switch to the inference tab to generate photos.)
```

**Comment by ultimatech-cn** [comment]
查到app.py，70行代码：
```
os.system(
        f'PYTHONPATH=. accelerate launch facechain/train_text_to_image_lora.py --pretrained_model_name_or_path={foundation_model_path} '
        f'--revision={revision} --sub_path="film/film" '
        f'--output_dataset_name={output_img_dir} --caption_column="text" --resolution=512 '
        f'--random_flip --train_batch_size=1 --num_train_epochs=200 --checkpointing_steps=5000 '
        f'--learning_rate=1e-04 --lr_scheduler="cosine" --lr_warmup_steps=0 --seed=42 --output_dir={work_dir} '
        f'--lora_r=32 --lora_alpha=32 --lora_text_encoder_r=32 --lora_text_encoder_alpha=32')
```

上一个版本是调用一个train_lora.sh的脚本
`sh_file_path = os.path.join('/'.join(os.path.abspath(__file__).split('/')[:-1]),
                                'train_lora.sh')`
发现错误原因，下面研究一下如何解决。

**Comment by ultimatech-cn** [comment]
自己解决了。修改了app.py的train_lora_fn函数，兼容window系统
    if platform.system() == 'Windows':
        command = [
            'accelerate', 'launch', 'facechain/train_text_to_image_lora.py',
            f'--pretrained_model_name_or_path={foundation_model_path}',
            f'--revision={revision}',
            '--sub_path=film/film',
            f'--output_dataset_name={output_img_dir}',
            '--caption_column=text',
            '--resolution=512',
            '--random_flip',
            '--train_batch_size=1',
            '--num_train_epochs=200',
            '--checkpointing_steps=5000',
            '--learning_rate=1e-04',
            '--lr_scheduler=cosine',
            '--lr_warmup_steps=0',
            '--seed=42',
            f'--output_dir={work_dir}',
            '--lora_r=32',
            '--lora_alpha=32',
            '--lora_text_encoder_r=32',
            '--lora_text_encoder_alpha=32'
        ]

        try:
            subprocess.run(command, check=True)
        except subprocess.CalledProcessError as e:
            print(f"Error executing the command: {e}")

**Comment by sunbaigui** [comment]
thanks for your contribution.

**Comment by sunbaigui** [comment]
welcome to join us, if you are interested. https://www.yuque.com/forms/share/5892d145-99c3-406c-8785-13dcb50353b8 .
